qchem/tests/test_dipole.py

```diff
@@ -0,0 +1,251 @@
+import os
+
+import numpy as np
+import pytest
+
+import pennylane as qml
+
+
+h2 = ["H", "H"]
+x_h2 = np.array([0.0, 0.0, -0.66140414, 0.0, 0.0, 0.66140414])
+coeffs_h2 = []
+coeffs_h2.append([0.0])
+coeffs_h2.append([0.0])
+coeffs_h2.append(
```

Comment:
Curious about the reason for constructing the list this way. Is it done to separate the coordinate components?

Reply:
Yes.